acceptance/suites/pre_suite/puppet3_compat/71_install_simmons_module.rb

```diff
@@ -1,4 +1,4 @@
-simmons_version = '0.3.1'
+simmons_version = '0.3.2'
```

Comment:
NOTE: This ticket will not be mergeable until dependent changes to the `nwolfe-simmons` module are merged and published in a new version, 0.3.2, to the Puppet Forge.  See https://github.com/nwolfe/puppet-simmons/pull/1.